src/controllers/confirm.company.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Request, Response } from "express";
+import { OBJECTIONS_COMPANY_PROFILE } from "../constants";
+import { Templates } from "../model/template.paths";
+import { getValueFromObjectionsSession } from "../services/objections.session.service";
+import logger from "../utils/logger";
+
+/**
+ * GET controller for check company details screen
+ * @param req
+ * @param res
+ * @param next
+ */
+
+export const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    if (req.session && req.session.data) {
+        const company: string = getValueFromObjectionsSession(req.session, OBJECTIONS_COMPANY_PROFILE);
+        return res.render(Templates.CONFIRM_COMPANY, {
+            company,
+            templateName: Templates.CONFIRM_COMPANY,
+        });
+    } else {
+        logger.error("Error displaying company data");
```

Comment:
could you use logger.errorRequest(req, ...    here, it will hopefully allow us to work out what function the error occurred in. Could you also put more info in the message, just thinking if this happens in live it won't help us much. Something like "Confirm Company - no company data found in session"

Reply:
Updated error logging